### ISSUE TITLE
Teampass database is not up to date - v3.1.2.115

### ISSUE THREAD

**Issue by gonzalodeniz** [nilsteampassnet/TeamPass#4368]
### Steps to reproduce
1.Upgrade from 3.1.2.90 to  3.1.2.115

### Expected behaviour
Succesful login

### Actual behaviour
Error loading index.php
![image](https://github.com/user-attachments/assets/5e5145a7-69e4-44c0-945d-b1202c64c5b2)

### Server configuration
Operating system: ubuntu 22.04

Web server: apache 2.4

Database: MariaDB10

PHP version: 8.2

Teampass version: 3.1.2.115

**Teampass configuration file:**

```
<?php
// DATABASE connexion parameters
define("DB_HOST", "teampass3_mariadb");
define("DB_USER", "teampass");
define("DB_PASSWD", "teampass");
define("DB_NAME", "teampass");
define("DB_PREFIX", "teampass_");
define("DB_PORT", "3306");
define("DB_ENCODING", "utf8");
define("DB_SSL", false); // if DB over SSL then comment this line
// if DB over SSL then uncomment the following lines
//define("DB_SSL", array(
//    "key" => "",
//    "cert" => "",
//    "ca_cert" => "",
//    "ca_path" => "",
//    "cipher" => ""
//));
define("DB_CONNECT_OPTIONS", array(
    MYSQLI_OPT_CONNECT_TIMEOUT => 10
));
define("SECUREPATH", "/var/www/html/teampass/includes");
define("SECUREFILE", "my-secret-file");

if (isset($_SESSION['settings']['timezone']) === true) {
    date_default_timezone_set($_SESSION['settings']['timezone']);
}
```

**Updated from an older Teampass or fresh install:**
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

**It does not exist. It has been deleted.**

#### Web server error log
```
Nothing
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
![image](https://github.com/user-attachments/assets/c8cca0d7-d935-496d-94eb-819f7d47a65e)

**Comment by gonzalodeniz** [comment]
It may have something to do with me renaming a folder as explained in issue 4365:
https://github.com/nilsteampassnet/TeamPass/issues/4365#issue-2548289458

```
mv install/install install/install_disable
```

But if I don't rename or delete it, an infinite loop occurs and the update doesn't finish.

**Comment by gonzalodeniz** [comment]
Thank you for your help @tomste 

Fixed in v3.1.2.120
